tests/integration/Settings/__snapshots__/Payments_Tab_Test__should_match_payment_tab_snapshots__stripe enabled__0.snapshot.html

```diff
@@ -18,7 +18,7 @@ <h2 class="tec-tickets__admin-settings-tickets-commerce-gateway-title">
 	<div class="tec-tickets__admin-settings-tickets-commerce-gateway-signup-links">
 		<div class="tec-tickets__admin-settings-tickets-commerce-gateway-signup-settings">
 	<div class="tec-tickets__admin-settings-tickets-commerce-gateway-connect-button">
-		<a data-gateway-onboard-complete="tecTicketsCommerceGatewayStripeSignupCallback" href="https://whodat.theeventscalendar.com/commerce/v1/stripe/connect?token=https://wordpress.test/index.php?rest_route=/tribe/tickets/v1/commerce/stripe/return&#038;return_url=https://whodat.theeventscalendar.com/commerce/v1/stripe/connected&#038;version={version}&#038;mode=live&#038;" data-gateway-button="true" id="connect_to_stripe" class="tec-tickets__admin-settings-tickets-commerce-gateway-connect-button-link">
+		<a data-gateway-onboard-complete="tecTicketsCommerceGatewayStripeSignupCallback" href="https://whodat.theeventscalendar.com/commerce/v1/stripe/connect?token=https://wordpress.test/index.php?rest_route=/tribe/tickets/v1/commerce/stripe/return&#038;return_url=https://whodat.theeventscalendar.com/commerce/v1/stripe/connected&#038;version={version}.1&#038;mode=live&#038;" data-gateway-button="true" id="connect_to_stripe" class="tec-tickets__admin-settings-tickets-commerce-gateway-connect-button-link">
```

Comment:
Shouldn't the added `.1` be part of the `{version}` replacement? Or is it actually something different?